packages/core/src/invoke.ts

```diff
@@ -60,8 +59,8 @@ export async function invokeWithMiddleware<TInput, TResult>(
 
       const result = await enhancedResolver(params, res.blitzCtx)
 
-      const duration = prettyMs(new Date().getTime() - startTime)
-      log.info(chalk.dim("Finished", "in", duration))
+      const duration = new Date().getTime() - startTime
+      log.info(chalk.dim(`Finished in ${duration}ms`))
```

Comment:
To remove pretty-ms, we need to replicate the full behavior. For example pretty-ms will convert to seconds if greater than 1000 ms.

Reply:
I'd say if you have handlers taking more than 1000 ms I'd say you have bigger problems

Reply:
Seems like an irrelevant optimization for me, but I can do it

Reply:
I reluctantly did it